### ISSUE TITLE
Accessibility: screen reader support is missing

### ISSUE THREAD

**Issue by vvillenave** [oguzhaninan/Buka#71]
Although Electron apps are generally lacking with regard to accessibility  
features, there are some ways of enabling some of these, for example  
when launching with the `--force-renderer-accessibility` switch.

Buka doesn’t have that ability, which makes it of no use for visually-impaired users.
